jenkins/run_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Install requirements.
+dnf -y install qemu-kvm systemd-container
+
+# Run a boot test.
+python3 -m unittest -v test.test_boot
```

Comment:
Are jenkins tests always run at root? And is the path to `run_tests` hard-coded, or can we throw in more tests which will then be run in parallel?